helion/_compiler/device_ir.py

```diff
@@ -230,6 +231,14 @@ def name(self) -> str:
 
     def codegen(self, state: CodegenState) -> list[object]:
         test = state.ast_arg(0)
+
+        test_proxy = state.proxy_arg(0)
+        if isinstance(test_proxy, torch.Tensor) and test_proxy.numel() == 1:
+            # Triton does not support `if one_elem_tensor:` but supports `if scalar:`,
+            # so we need to use tl.sum to extract the scalar.
+            test_code = ast.unparse(test)
```

Comment:
if we can guarantee that `if` is always followed by a simple variable name (and not expression), then we can simplify this to just use `test.id`.